content/en/docs/tasks/traffic-management/egress/egress-gateway-tls-origination/index.md

```diff
@@ -635,20 +635,20 @@ to hold the configuration of the NGINX server:
     you used for generating your `istio.yaml`:
 
     {{< text bash >}}
-    $ helm template install/kubernetes/helm/istio/ --name istio --namespace istio-system -x charts/gateways/templates/deployment.yaml --set gateways.istio-ingressgateway.enabled=false \
-    --set gateways.istio-egressgateway.enabled=true \
-    --set 'gateways.istio-egressgateway.secretVolumes[0].name'=egressgateway-certs \
-    --set 'gateways.istio-egressgateway.secretVolumes[0].secretName'=istio-egressgateway-certs \
-    --set 'gateways.istio-egressgateway.secretVolumes[0].mountPath'=/etc/istio/egressgateway-certs \
-    --set 'gateways.istio-egressgateway.secretVolumes[1].name'=egressgateway-ca-certs \
-    --set 'gateways.istio-egressgateway.secretVolumes[1].secretName'=istio-egressgateway-ca-certs \
-    --set 'gateways.istio-egressgateway.secretVolumes[1].mountPath'=/etc/istio/egressgateway-ca-certs \
-    --set 'gateways.istio-egressgateway.secretVolumes[2].name'=nginx-client-certs \
-    --set 'gateways.istio-egressgateway.secretVolumes[2].secretName'=nginx-client-certs \
-    --set 'gateways.istio-egressgateway.secretVolumes[2].mountPath'=/etc/nginx-client-certs \
-    --set 'gateways.istio-egressgateway.secretVolumes[3].name'=nginx-ca-certs \
-    --set 'gateways.istio-egressgateway.secretVolumes[3].secretName'=nginx-ca-certs \
-    --set 'gateways.istio-egressgateway.secretVolumes[3].mountPath'=/etc/nginx-ca-certs > \
+    $ istioctl manifest template --set values.gateways.istio-ingressgateway.enabled=false \
+    --set values.gateways.istio-egressgateway.enabled=true \
+    --set 'values.gateways.istio-egressgateway.secretVolumes[0].name'=egressgateway-certs \
```

Comment:
I don't know if this will work with arrays - needs testing.  @ostromart thoughts?

Reply:
That's a very good point. It will not work, we have to pass it as a yaml file. In this case I think this is a good idea anyway given the length of this command.

Reply:
@ostromart can you also change this one to a correct command in a followup PR?

Reply:
I've created https://github.com/istio/istio.io/issues/5275 to track it.